COPYING

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2003-2006 Eelco Dolstra
+Copyright (c) 2003-2006 Eelco Dolstra, Google Inc.
```

Comment:
(I’m quite busy, so it’s a hurried reply, sorry)

@eduarrrd If you created that in your free time, it doesn’t count as [“work for hire” (link: Wikipedia)][wfh] even under US copyright.

To quote [Columbia Law School’s article][cls] on this topic:

> On the other hand, if you’ve written a novel or a computer application in your spare time, and that isn’t your job, you, not your employer, own those rights. That’s important to remember for the many creators who also have day jobs.

In any case you would be entitled to moral rights under the Berne convention, but *if* you assigned the rights to the work you did here to Google (e.g. as work for hire or a separate contract), it might indeed be that Google has to be listed as a copyright holder.

Without seeing the details, the situation you describe here, it indeed sounds like you wrote this in your free time and Google has no right to be listed as copyright holder. The one who has the right to be listed is you, who wrote it.

As @zimbatm noticed, all the authors should have been listed in the copyright headers and the `COPYING` or `AUTHORS` file already before. So that needs to be fixed at some point in any case.


[wfh]: https://en.wikipedia.org/wiki/Work_for_hire
[cls]: http://web.law.columbia.edu/keep-your-copyrights/copyrights/works-made-hire